msticpy/analysis/code_cleanup.py

```diff
@@ -0,0 +1,71 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""Code cleanup functions to re-format obfuscated code."""
+import re
+
+from .._version import VERSION
+
+__version__ = VERSION
+__author__ = "Ian Hellen"
+
+
+def format_powershell(code: str) -> str:
```

Comment:
Great idea - would be good to extend in future to cover some of the other obsfucation techniques in https://github.com/danielbohannon/Invoke-Obfuscation